src/constants/styling/buttons.scss

```diff
@@ -33,6 +35,69 @@
   font-weight: 600;
   padding: 5px 15px;
   border-radius: 5px;
+  // margin-bottom: 3px;
+  // margin-left: 0.5rem;
+
+  &.darker {
+    color: $NileBlue;
+    border: 1px solid $NileBlue;
+  }
+
+  &:not(:disabled):hover {
+    opacity: 0.8;
+  }
+
+  &:disabled {
+    opacity: 0.5;
+    cursor: not-allowed;
+  }
+
+  &:focus {
+    outline: none;
+    border: 1px solid $Malibu;
+  }
+}
+
+.primaryButton1 {
+  background: none;
+  letter-spacing: 1px;
+  font-weight: 600;
+  padding: 5px 15px;
+  text-decoration: none;
+  border: 1px solid $RoyalBlue;
+  background-color: $RoyalBlue;
+  color: $Cararra;
+  cursor: pointer;
+  border-radius: 5px;
+  margin-right: 9px;
+  margin-left: 0.5rem;
```

Comment:
This works but it could be DRYer (don't repeat yourself). Could you minimise repeating code by using

&.1 {
  margin-right: 9px;
  margin-left: 0.5rem;
}

(or even better give them more obvious names that 1 e.g. navbutton